app/controllers/api/v1/votes_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::V1::VotesController < ApplicationController
+  protect_from_forgery with: :null_session
+  protect_from_forgery unless: -> { request.format.json? }
+  before_action :authorize_user
+
+  def create
+    user = current_user
+    review = Review.find(params[:review_id])
+    vote = Vote.new
+    vote.helpful = params[:helpful]
+    vote.user = user
+    vote.review = review
+    vote.save
+    render json: {
+      vote: vote,
+      reviews: serialized_reviews
+    }
+  end
+
+  def update
+    vote = Vote.find(params[:id])
+    vote.helpful = params[:helpful]
+    vote.save
+    render json: {
+      votes: Vote.by_current_user(current_user),
+      reviews: serialized_reviews
+    }
+  end
+
+  def destroy
+    vote = Vote.find(params[:id])
+    vote.delete
+    render json: {
+      votes: Vote.by_current_user(current_user),
+      reviews: serialized_reviews
+    }
+  end
```

Comment:
all of these render json statements could be refactored into something like, 
```
render json: Review.find(params[:review_id]).restaurant.reviews
```

Reply:
we can't have the votes called within the review serializer, because the votes will then be nested within each review